tests/integration/dogshell/test_dogshell.py

```diff
@@ -124,7 +124,7 @@ def test_metrics(self):
 
         # Give the host some tags
         tags0 = ["t0", "t1"]
-        out, _, _ = self.dogshell_with_retry(["tag", "add", host] + tags0)
+        out, _, _ = self.dogshell_with_retry(["tag", "add", host] + tags0, retry_limit=15)
```

Comment:
Should we just bump that to 30 directly ? That endpoint was really flaky, I'm not sure that 5 seconds bump is going to help a lot

Reply:
Yeah thats fine, I don't really see harm in bumping. Since I couldn't reproduce this locally, I was going to bump and just monitor the CI runs. Bumping (with comment)